src/const_choice.rs

```diff
@@ -154,6 +193,12 @@ impl ConstChoice {
     pub(crate) const fn to_u8(self) -> u8 {
         (self.0 as u8) & 1
     }
+
+    /// WARNING: this method should only be used in contexts that aren't constant-time critical!
+    #[inline]
+    pub(crate) const fn to_bool(self) -> bool {
```

Comment:
Maybe it's worth suffixing with `vartime`

Reply:
Done in [47c4db3](https://github.com/RustCrypto/crypto-bigint/pull/628/commits/47c4db32624b52933dafa080dc585c76e5deb941)